app/models/concerns/thredded/user_topic_read_state_common.rb

```diff
@@ -9,7 +9,7 @@ module UserTopicReadStateCommon
 
     # @return [Boolean]
     def read?
-      postable.updated_at <= read_at
+      postable.updated_at.to_i <= read_at.to_i
```

Comment:
@jayroh This is strange, the times should be identical, because `read_at` is set to `updated_at` of the last post.